plone-and-python-community.rst

```diff
@@ -492,9 +492,9 @@ Technology
 
 .. note::
 
-    It is a question of Adaptation
+    It is a question of adaptation
 
-    in Business often called the Selection of Best of Breed
+    In business this is often called Best of Breed selection
```

Comment:
Adaptation isn't really the same as "best of breed", which is (I think) more the idea that for every component of a big system you pick whichever thing is best for the component, and then you end up with a mix of things.